index.html

```diff
@@ -95,13 +95,13 @@ <h2>$49</h2>/month
                 <div class="foter_block_end_1">
                     <div class="foter_block_end_img"></div>
                     <div class="foter_block_end_social-icon">
-                        <div><i class="fab fa-facebook color-icon"></i></div>
-                        <div><i class="fab fa-twitter color-icon"></i></div>
-                        <div><i class="fab fa-google color-icon"></i></div>
+                        <div ><a href="/jpg/mini_2.png"><i class="fab fa-facebook color-icon"></i></a></div>
```

Comment:
please change to correct url . for example for facebook => https://facebook.com etc.

Reply:
done